app/persistence/transaction/transaction.go

```diff
@@ -165,15 +166,17 @@ func (tr *Repository) GetSkippedOrInitialTransactionsAndMessages() (map[transact
 }
 
 // Create creates new record of Transaction
+//TODO: Should ExecuteEthTransaction field be added to the other repository functions
```

Comment:
I think that the only other place to add `ExecuteEthTransaction` is in the `SaveRecoveredTxn` function. We should add the property there. It would work out of the box